.github/CODEOWNERS

```diff
@@ -4,4 +4,5 @@
 # the repo. Unless a later match takes precedence, members of
 # @rewindio/codeowners will be requested for review when someone
 # opens a pull request.
-*       @MarcMorinville-Rewind @jhuff078 @smandegar
+
+* @smandegar @julesclay @mkhinda29
```

Comment:
Why don't we just set this to the developers group?  I don't think it needs to be individuals does it?